admin/javascript/LeftAndMain.Panel.js

```diff
@@ -134,5 +134,14 @@
 				return false;
 			}
 		});
+
+		$('.cms-content-tools').entwine({
```

Comment:
Any reason why you’re using the `if` statement instead of just binding to `.cms-content-tools.collapsed` here?